### PR TITLE
adding coords back to footer in ui

### PR DESCRIPTION
**Pull request by jackgrossman18**
@brianhatchl 

adding coords back to ui within the footer

ref #1318

### CODE REVIEW

File: modules/ui/init.js [hunk 1]
```diff
@@ -249,6 +249,11 @@ export function uiInit(context) {
                 .call(uiAccount(context));
         }
 
+        aboutList
+            .append('li')
+            .attr('class', 'info-block')
+            .call(uiCoordinates(context));
+
```
Comment (brianhatchl) [inline modules/ui/init.js]: Looking at the bottom bar in the UI, I think the coordinate display would be more at home next to the scale bar on the left.  It also has the behavior of changing units when clicked.  I do like this better than the right-click menu, but please add a tool tip like in Hoot1, eg. "Click to change coordinate format".  Also I think MGRS should be one of the supported formats.

File: modules/ui/coordinates.js [hunk 2]
```diff
@@ -143,8 +148,17 @@ export function uiCoordinates(context) {
         return zone.toString() + sn + ' ' + Xutm.toString() + 'm E ' + Yutm.toString() + 'm N';
     }
 
+    function invaildCoordinates(coords) {
```
Comment (brianhatchl) [inline modules/ui/coordinates.js]: typo in "invalid"

**Comment by jackgrossman18** [comment]
@brianhatchl moved the coords to the left of the footer, added a popup window like hoot 1, and included MGRS as an option

**Comment by brianhatchl** [comment]
Did some more clean up as seen in commit messages

**Pull request merged**